apps/app/src/lib/data/agents.ts

```
@@ -0,0 +1,60 @@
+  id: string;
+  agentName: string;
+  models: string[];
+  branches: string[];
+};
+
+export type CreateAgentData = {
+  agentName: string;
+  models: string[];
+};
+
+export type CreateAgentResponse = Agent;
```

ESLint

[error] Remove this redundant type alias and replace its occurrences with "Agent".

(sonarjs/redundant-type-aliases)